modules/styles.py

```diff
@@ -36,6 +38,16 @@ def apply_styles_to_prompt(prompt, styles):
 
     return prompt
 
+class SDcustom(Dialect):
+        """Describe the usual properties of Excel-generated CSV files."""
+        delimiter = ','
+        quotechar = '"'
+        doublequote = True
+        skipinitialspace = True
+        lineterminator = '\r\n'
+        escapechar = '\''
+        quoting = QUOTE_MINIMAL

```

Comment:
```suggestion
class SDcustom(excel):
    skipinitialspace = True
```
Instead of copying the `excel` class, extend it. That way you only need the properties that you're changing.